packages/start-slicemachine/src/lib/setupSentry.ts

```diff
@@ -0,0 +1,70 @@
+import * as Sentry from "@sentry/node";
+
+import semver from "semver";
+import { App, H3Event, sendError } from "h3";
+import { SliceMachineManager } from "@slicemachine/manager";
+import { SENTRY_EXPRESS_DSN } from "../constants";
+
+export const setupSentry = async (
+	manager: SliceMachineManager,
+	app: App,
+): Promise<void> => {
+	!isDevEnv() || process.env.SENTRY_AUTH_TOKEN !== undefined;
+	if (!isSentryEnabled()) {
+		return;
+	}
+
+	const sliceMachineVersion =
+		await manager.versions.getRunningSliceMachineVersion();
+
+	let user;
```

Comment:
We're missing a type annotation here:

```ts
let user: PrismicUserProfile | undefined;
```

The type is exported from `@slicemachine/manager`:

```ts
import { PrismicUserProfile } from "@slicemachine/manager";
```

(Small nitpick, but since the value is the user's profile, not necessarily the "user", could we rename the variable to `profile` or `userProfile`?)